dev/tasks/build-images

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
```

Comment:
I don't have any strong opinion, but we now have scripts with `.sh` extension :)

Reply:
These are the tasks where I have ideas :-)